rust/stub/src/pe_section.rs

```diff
@@ -6,25 +6,25 @@
 
 use alloc::{borrow::ToOwned, string::String};
 
-/// Extracts the data of a section of a PE file.
-pub fn pe_section<'a>(file_data: &'a [u8], section_name: &str) -> Option<&'a [u8]> {
-    let pe_binary = goblin::pe::PE::parse(file_data).ok()?;
+/// Extracts the data of a section of a loaded PE file.
+pub fn pe_section<'a>(pe_data: &'a [u8], section_name: &str) -> Option<&'a [u8]> {
+    let pe_binary = goblin::pe::PE::parse(pe_data).ok()?;
 
     pe_binary
         .sections
         .iter()
-        .find(|s| s.name().unwrap() == section_name)
+        .find(|s| s.name().map(|n| n == section_name).unwrap_or(false))
         .and_then(|s| {
-            let section_start: usize = s.pointer_to_raw_data.try_into().ok()?;
+            let section_start: usize = s.virtual_address.try_into().ok()?;
```

Comment:
This is the change in section parsing that is required when we don't parse a file anymore.